.bazelrc

```diff
@@ -15,6 +16,9 @@ build --action_env=CARGO_BAZEL_REPIN=true
 test --test_output=streamed
 test --nocache_test_results
 test --action_env=RUST_TEST_THREADS=1
+build:linux --cxxopt="-std=c++17"
+build:windows --cxxopt="/std:c++17"
+build:macos --cxxopt="-std=c++17"
```

Comment:
Is the a reason for this duplication?

Reply:
oops, my bad, i want to add it for test actually here.
```diff
-build:linux --cxxopt="-std=c++17"
+test:linux --cxxopt="-std=c++17"
```

Reply:
@elBoberido changed in the newest commit, thanks

Reply:
oh, found a refinement here because test inherits the build so as long as the build one is not removed, we can remove the cxxopt setting for test.

> The following commands inherit from (and are more specific than) build: test, run, clean, mobile-install, info, print_action, config, cquery, and aquery